cmd/goose/main.go

```diff
@@ -219,6 +220,7 @@ Examples:
     goose mssql "sqlserver://user:password@dbname:1433?database=master" status
     goose clickhouse "tcp://127.0.0.1:9000" status
     goose vertica "vertica://user:password@localhost:5433/dbname?connection_load_balance=1" status
+    goose ydb "grpcs://localhost:2135/local?go_query_mode=scripting&go_fake_tx=scripting&go_query_bind=declare,numeric" status
```

Comment:
Can you drop a link to the docs or a comment on what these are?

```
go_query_mode=scripting
go_fake_tx=scripting
go_query_bind=declare,numeric
```

Reply:
`go_query_mode` - it is required because YDB have different methods for different types of queries (DDL, DML, etc.). [Docs](https://github.com/ydb-platform/ydb-go-sdk/blob/master/SQL.md#query-modes)
`go_fake_tx` - spetial mode for external tools like goose, which requires a transactional DDL. Release in [v3.44.0](https://github.com/ydb-platform/ydb-go-sdk/releases/tag/v3.44.0)
`go_query_bind=declare,numeric` - released in [v3.44.0](https://github.com/ydb-platform/ydb-go-sdk/releases/tag/v3.44.0) and published in article [database/sql bindings for YDB in Go](https://medium.com/ydbtech/database-sql-bindings-for-ydb-in-go-a8a2671a8696)